README.md

```diff
@@ -1,7 +1,10 @@
 ![GitHub issues](https://img.shields.io/github/issues/modedevteam/mode-community)
 ![Follow](https://img.shields.io/twitter/follow/modedevteam?style=social)
 [![Discord](https://img.shields.io/badge/Discord-Join%20Us-blue)](https://discord.gg/FRDxms57pG)
-![Early Access](https://img.shields.io/badge/status-early%20access-007FFF)
+[![Early Access](https://img.shields.io/badge/status-early%20access-007FFF)](https://marketplace.visualstudio.com/items?itemName=aruna-labs.mode)
+[![VSCode Extension Installs](https://img.shields.io/vscode-marketplace/i/aruna-labs.mode.svg)](https://marketplace.visualstudio.com/items?itemName=aruna-labs.mode)
+![VSCode Extension Downloads](https://img.shields.io/vscode-marketplace/d/aruna-labs.mode.svg)
+![VSCode Extension Rating](https://img.shields.io/vscode-marketplace/r/aruna-labs.mode.svg)
```

Comment:
This is overall awesome, but I see this issue with rating. Let's either skip including the rating or ideally, resolve the issue and push:

![image](https://github.com/user-attachments/assets/9bfa4c48-915e-464f-90bd-1135ce96b343)

Reply:
![CleanShot 2024-12-24 at 00 33 58](https://github.com/user-attachments/assets/2354dad3-f02d-44ce-966b-98495cd10649)

I don't think rate limiting is isolated only to this particular badge. Also, as seen by above screenshot it's not always presented.